client/components/App.vue

```diff
@@ -92,7 +92,7 @@
 </template>
 
 <script>
-import {throttle} from "lodash";
+const throttle = require("lodash/throttle");
```

Comment:
Doesn't `import { throttle } from 'lodash/throttle'` work?

Reply:
Right, that exact wording would work.